### PR TITLE
Exercise 5

### PR DESCRIPTION
**Pull request by Stuart-Knowles**
Here's my exercise 5 submission.

Just to note I didn't do the dockerised development with a bind mount, but with docker compose develop instead. If I were doing this properly and using, e.g., gunicorn not flask for serving, then I'd do it in the dockerfile not docker compose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,73 @@
...
+ * Restarting with stat
+ * Debugger is active!
+ * Debugger PIN: 113-666-066
+```
+Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running in a container
+To run the app in a docker container, run:
+```
+docker compose up app
```
Comment (JackMeadDev) [inline README.md]: Unless I'm misunderstanding, I think a mismatched name here:
```suggestion
docker compose up todo-app
```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: ultimate-todo-app-of-ultimate-destiny
```
Comment (JackMeadDev) [inline docker-compose.yml:1]: Love it 👍

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Dockerfile
+docker-compose.yml
+deployment/
+.*
```
Comment (JackMeadDev) [inline .dockerignore]: Nice, we might also want to add any caching files, e.g.:
```suggestion
.*
**/__pycache__
**/.pytest_cache
```
(Note that `dockerignore` doesn't use _precisely_ the same syntax as `gitignore`, so we need to specify that those directories should be ignored regardless of arbitrary nesting)

**Review by JackMeadDev — APPROVED** [review]
Lovely, I had no problems running your containerised app bar one instruction typo, the tests all just worked, and nice use of newer features than we discuss! 😄 Feel free to go ahead and merge when you're ready

File: README.md (hunk 1 above)
Comment (Stuart-Knowles) [reply README.md]: Ooh good catch, thanks!

**Pull request merged**